conans/client/installer.py

```diff
@@ -161,9 +161,9 @@ def _build_package(self):
         self._conan_file.install_folder = self.build_folder
 
         # Read generators from conanfile and generate the needed files
-        logger.debug("Writing generators")
+        logger.info("GENERATORS: Writing generators")
         write_generators(self._conan_file, self.build_folder, self._out)
-        logger.debug("Files copied after generators %s", os.listdir(self.build_folder))
+        logger.debug("GENERATORS: Files copied after generators %s", os.listdir(self.build_folder))
```

Comment:
Are these generators files or other kind of files? The "Files copied after generators" seems misleading. Maybe "files created %s" to make clear those are files created by generators, not other kind of files